openapi.yaml

```diff
@@ -543,6 +552,8 @@ paths:
                   type: object
                 app_metadata:
                   type: object
+                channel:
+                  type: string
```

Comment:
Replicate the enum on input types only (request bodies) but not on response bodies.